docs/pages/plugins.md

```diff
@@ -4,10 +4,14 @@
 
 Current official plugins:
 
-- npm - publish code to npm (DEFAULT)
-- chrome - publish code to Chrome Web Store
+- Chrome - publish code to Chrome Web Store
```

Comment:
```suggestion
- chrome - publish code to Chrome Web Store
```

Maybe the plugin name shouldn't be capitalized to be consistent with all the others.